spec/controllers/todos_controller_spec.rb

```diff
@@ -1,17 +1,57 @@
 require "spec_helper"
 
 RSpec.describe TodosController, type: :controller do 
-	# subject(:todo) { create(:todo) }
-	describe "GET /index" do
-		let(:todo) { create(:todo) }
-		it "returns a todo list" do
-			todo
-			get :index, format: :json
-			expect(response).to have_http_status(:ok)
-			todo_json = JSON.parse(response.body)
-			expect(todo_json.first["title"]).to eq("Go shopping")
-		end
-	end
+  describe "GET /index" do
+    let(:todo) { create(:todo) }
+    it "returns a todo list" do
+      todo
+      get :index, format: :json
+      expect(response).to have_http_status(:ok)
+      todo_json = JSON.parse(response.body)
+      expect(todo_json.first["title"]).to eq("Go shopping")
+    end
+  end
 
-	
+  describe "POST /create" do
+    context "with valid params" do
+      let(:valid_params) { attributes_for(:todo, title: "Go shopping") }
+      it "creates a todo list" do
+        post :create, todo: valid_params, format: :json
+        expect(response).to have_http_status(:created)
+        todo_json = JSON.parse(response.body)
+        expect(todo_json["title"]).to eq("Go shopping")
+      end
+    end
+    context "with invalid params" do
+      let(:invalid_params) { attributes_for(:todo, title: nil) }
+      it "does not create a todo list" do
+        post :create, todo: invalid_params, format: :json
+        expect(response).to have_http_status(422)
+        todo_json = JSON.parse(response.body)
```

Comment:
So here, rather than inspecting the `todo` object -- which just failed to save -- it'd be better to assert something about the error message. Maybe, like `expect(todo_json["errors"])` something something something.